### DISCUSSION TITLE
change music program (instrument)

### DISCUSSION THREAD

**Discussion by sugizo** [Rainbow-Dreamer/musicpy#63] [Q&A]
how to change music program (instrument) ?

thanks and best regards

**Comment by Rainbow-Dreamer** [comment]
Hi, you can read musicpy's wiki about how to change music program, here is the link to the chapter: https://github.com/Rainbow-Dreamer/musicpy/wiki/Basic-syntax-of-piece-type#quickly-change-the-instruments-of-tracks-of-the-piece-type
